### PR TITLE
[765]: signpost user when mentor doesn't have a trn

### PR DESCRIPTION
**Pull request by ltello**
[Issue](https://github.com/DFE-Digital/register-early-career-teachers/issues/765)

https://github.com/user-attachments/assets/9352e826-fbf1-4213-a6fd-5285984d8dac

### CODE REVIEW

File: app/wizards/schools/register_mentor/cannot_without_trn_step.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module Schools
+  module RegisterMentor
+    class CannotWithoutTRNStep < Step
```
Comment (TobyRet) [inline app/wizards/schools/register_mentor/cannot_without_trn_step.rb:3]: Cannot (do what?) without a TRN? 
Register?

Could be worth renaming to make this clear

File: app/wizards/schools/register_mentor/wizard.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@ class Wizard < DfE::Wizard::Base
       steps do
         [
           {
+            cannot_without_trn: CannotWithoutTRNStep,
```
Comment (TobyRet) [inline app/wizards/schools/register_mentor/wizard.rb:9]: https://github.com/DFE-Digital/register-early-career-teachers/pull/869#discussion_r1867702676

File: app/wizards/schools/register_mentor/cannot_without_trn_step.rb (hunk 1 above)
Comment (ltello) [reply app/wizards/schools/register_mentor/cannot_without_trn_step.rb:3]: Happy to change it but is it not too verbose given that we are inside the `Schools::RegisterMentor` namespace?

Comment (ltello) [reply app/wizards/schools/register_mentor/cannot_without_trn_step.rb]: Renamed to NoTRNStep

**Review by TobyRet — APPROVED** [review]

**Pull request merged**